README.md

```diff
@@ -270,9 +270,10 @@ in a bit.)
 
 ### Opening our project in VS Code
 
-Now we'll want to open VS Code and see how we can use it to edit and run our code.
-Launch VS Code, and then choose `File -> Open…`. Navigate to your clone of the repo
-and choose `Open`.
+Now we'll want to open VS Code (search for "Visual Studio Code" in the application
```

Comment:
Do we still want the comment about the application menu? Does it make sense of they're running on their own gear?

Reply:
Yeah that's pretty specific to Gnome. We should assume students know how to open applications on whatever system they're using.

Reply:
I've gone ahead and changed that and I'll go ahead and merge this in.